python/lsst/ip/isr/deferredCharge.py

```diff
@@ -941,6 +951,14 @@ def fromTable(cls, tableList):
 
             inDict['serialTraps'][amp] = ampTrap
 
+        # Version check
+        if calibVersion < 1.1:
+            # This version might be in the wrong units (not electron),
+            # and does not contain the gain information to convert
+            # into a new calibration version.
+            raise RuntimeError(f"Using old version of CTI calibration (ver. {calibVersion} < 1.1), "
+                               "which is no longer supported.")
```

Comment:
Did we decide that this is okay that we are abandoning any cti calibs out in the wild?  (Are there any?)

Reply:
I think it is just me. By default, the IsrTask turns off deferred charge correction, and I don't think that there are any curated CTI calibs for any instrument in /repo/embargo, /repo/main, /repo/ir2, or /repo/ops-rehearsal-3-prep.

Reply:
I don't think there are any CTI calibrations in any repo that are more than proof of concepts.  There are certainly none linked to the main calibration collections.

Reply:
Great!  Then I consider this resolved.